src/components/AddLinkForm.tsx

```diff
@@ -0,0 +1,14 @@
+import style from '../styles/AddLinkForm.module.css';
+
+const AddLinkForm = () => {
+  return(
+    <form className={`${style.form}`}>
```

Comment:
문자열 타입의 값을 그대로 넣으셔도 됩니다
```suggestion
    <form className={style.form}>
```